Libraries/LibWeb/Dump.cpp

```diff
@@ -61,6 +65,46 @@ static void indent(StringBuilder& builder, int levels)
         builder.append("  "sv);
 }
 
+static String s_debug_dump_path;
+
+void set_debug_dump_path(StringView path)
+{
+    s_debug_dump_path = String::from_utf8(path).release_value_but_fixme_should_propagate_errors();
+}
+
+static void dump_to_file(StringBuilder& builder, StringView filename)
+{
+    ByteString dump_path = s_debug_dump_path.to_byte_string();
```

Comment:
To reiterate, just using `Core::StandardPaths::tempfile_directory()` here would have been fine with me.